app/components/CalendarSubscriptionLinks.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+interface CalendarSubscriptionLinksProps {
+	webcalUrl: string;
+}
+
+export function CalendarSubscriptionLinks({
+	webcalUrl,
+}: CalendarSubscriptionLinksProps) {
+	return (
+		<div>
+			{[
+				{ href: webcalUrl, vendor: "Apple" },
```

Comment:
It would be nice to have something here for people who don't use Apple/Google calendars (like me 😃)

Reply:
We can potentially integrate with any service that offers ics support. What do you suggest?